test/tests/audioContext.js

```diff
@@ -0,0 +1,20 @@
+let { getAudioContext } = p5.prototype;
+let { expect } = chai;
+describe('Audiocontext', function () {
+  it('is Running ', async function () {
+    let audioContext = await getAudioContext();
```

Comment:
I'm not sure if we get many benefits from these tests since `getAudioContext` returns a Web Audio API `AudioContext`. We are testing the Web Audio API, and I think our tests should instead focus on the p5.sound API. For example, we could simply have a test for the `getAudioContext` method. Not sure what else we're really able to do with this unit since userSartAudio depends on user interaction...

Reply:
getAudioContext() Is one of p5sounds APIs ,
Although it returns web audio node instance but still we are confirming here that our given APIs is working fine !!